frontend/packages/dev-console/src/components/multi-tab-list/MultiTabListPage.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { match as Rmatch } from 'react-router-dom';
+import * as _ from 'lodash';
+import {
+  history,
+  PageHeading,
+  HorizontalNav,
+  Page,
+  Dropdown,
+} from '@console/internal/components/utils';
+import { ALL_NAMESPACES_KEY } from '@console/shared';
+import { referenceForModel } from '@console/internal/module/k8s';
+import NamespacedPage, { NamespacedPageVariants } from '../NamespacedPage';
+import { MenuActions, MenuAction } from './multi-tab-list-page-types';
+
+interface MultiTabListPageProps {
+  title: string;
+  badge?: React.ReactNode;
+  menuActions?: MenuActions;
+  pages: Page[];
+  match: Rmatch<any>;
+}
+
+const MultiTabListPage: React.FC<MultiTabListPageProps> = ({
+  title,
+  badge,
+  pages,
+  menuActions,
+  match,
+}) => {
+  const {
+    params: { ns },
+  } = match;
+  const multiTabListPageTitle = (
+    <span style={{ display: 'flex', alignItems: 'flex-end' }}>
+      {title}
+      <span style={{ marginLeft: 'var(--pf-global--spacer--md)' }}>{badge}</span>
+    </span>
+  );
```

Comment:
Yeah wanted to keep this so that it is aligned with what is shown in UXD. There are a couple of differences I see, spacing, alignment. I can remove it if that makes more sense.

Reply:
If you're unsure, we can definitely rope UX in to discuss this change. Personally I feel this is not a requirement given we are re-using components here and we should avoid making that inconsistency unless it's explicitly a direction we are migrating to.

Reply:
Yeah, anyways I removed it :stuck_out_tongue:. Makes sense to keep it consistent.

Reply:
Updated the SS.

Reply:
I will tag UX in the PR for actually adding the pages and there we can take there approval.